### DISCUSSION TITLE
My school blocked javascript on webstorex

### DISCUSSION THREAD

**Discussion by 1-Mythical** [3kh0/ext-remover#280] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I heard LTbeef stopped working after 106 on chrome, but it works just fine for me and I am on 107.

Anyways, so the title says it all. What should I do?

**Comment by skoopy-star** [comment, marked as answer]
use a different site like webstorefjafsh

**Reply by 1-Mythical** [reply]
This does not work. They blocked chrome.google.com/webstore. any text after that will block javascript. In fact, the webstore is just a blank page. Sorry for being late to respond.

**Reply by nicolasog12** [reply]
then you can't do it.
simple as that.

**Comment by Aka-but-nice** [comment]
Is it an extension block or does it say this site is blocked or sum
